.github/ISSUE_TEMPLATE/10-issue_report.yaml

```diff
@@ -0,0 +1,136 @@
+name: Report a Bug
+title: "<title>"
```

Comment:
```suggestion
title: "[BUG]: "
```

Reply:
I would have neither. The pre-filled title must be manually removed by the user of the form, which is not nice. Also I wouldn't prefix the titles, as we have labels for that kind of stuff.